web/src/server_events.js

```diff
@@ -239,7 +241,9 @@ function get_events({dont_block = false} = {}) {
                 blueslip.error("Failed to handle get_events error", undefined, error);
             }
 
-            const retry_delay_secs = util.get_retry_backoff_seconds(xhr, get_events_failures);
```

Comment:
Why not just move this to the top of the function? `util.get_retry_backoff_seconds` does not have side effects, so I don't think there's a reason to have this logic to set its value twice with the same value...

Reply:
Yes, makes sense, I have removed that previous commit, and just moved the `util.get_retry_backoff_seconds` just before the first place where it is needed.